src/components/StudyInfo/CavaticaCounts.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import {List, Icon, Popup, Label, Button} from 'semantic-ui-react';
+
+const CavaticaCounts = ({projects, title}) => {
+  if (projects && projects.length > 0) {
+    const projectTypes = {DEL: 'Delivery', HAR: 'Analysis', RES: 'Research'};
+    const types = projects.map(({node: {projectType}}) => projectType);
+    const typeCounts = types.reduce((count, type) => {
+      count[type] = (count[type] || 0) + 1;
+      return count;
+    }, {});
+
+    return (
+      <List horizontal>
+        <List.Item
+          as={Link}
+          to={`/study/${title}/cavatica`}
+          onClick={e => e.stopPropagation()}
+        >
+          <Icon name="code branch" />
+          {projects.length} projects
+        </List.Item>
+        {Object.keys(projectTypes).map(
+          type =>
+            type in typeCounts && (
+              <Popup
+                inverted
+                position="top center"
+                size="small"
+                content={projectTypes[type]}
+                key={type}
+                trigger={
+                  <List.Item>
+                    <Label circular empty size="mini" color="olive" />{' '}
+                    {typeCounts[type]}
+                  </List.Item>
+                }
+              />
+            ),
+        )}
+      </List>
+    );
+  } else {
+    return (
+      <List horizontal>
+        <List.Item
+          as={Link}
+          to={`/study/${title}/cavatica`}
+          onClick={e => e.stopPropagation()}
+        >
+          <Icon color="red" name="code branch" />
+          <Button.Group size="mini" compact basic>
+            <Button primary size="mini" icon="linkify" content="LINK" />
```

Comment:
each button needs to link directly to the page and have the appropriate modal open. This could possibly be done with a react hook that reads a url hash (i.e. `#link-cavatica-project`) that sets the open state for the modal